third_party/move/move-model/src/ast.rs

```diff
@@ -626,11 +626,14 @@ pub enum ExpData {
     Return(NodeId, Exp),
     /// Represents a sequence of effects, the last value also being the result.
     Sequence(NodeId, Vec<Exp>),
-    /// Represents a loop, with a body expression.
```

Comment:
The previous comment was useful for newbs looking at this code.

Reply:
I feel its a good excercise. If the reader is not following the the Exp is the body, there is little hope. I forgot why this was changed, but I think it is as good as before.